cms-editor-test/config/users.xml

```diff
@@ -1,2 +1,13 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<userConfig/>
+<userConfig>
+    <user>
+        <username>testUser1</username>
+        <password>123456</password>
+        <role>CMS_ADMIN</role>
+    </user>
+    <user>
+        <username>testUser2</username>
+        <password>123456</password>
+        <role>CMS_USER</role>
```

Comment:
Could you change the name of the test users?
E.g. testUser1 -> cmsAdmin, testUser -> normalUser
Use the Everybody role for the testUsers2.